src/palette/internal/palettecreator.cpp

```diff
@@ -214,7 +214,7 @@ PalettePtr PaletteCreator::newBeamPalette()
 PalettePtr PaletteCreator::newDynamicsPalette(bool defaultPalette)
 {
     PalettePtr sp = std::make_shared<Palette>(Palette::Type::Dynamic);
-    sp->setName(QT_TRANSLATE_NOOP("palette", "Dynamics"));
+    sp->setName(QT_TRANSLATE_NOOP("palette", "Dynamics & Hairpins"));
```

Comment:
as that is what the palette contains...

Reply:
I think that when we moved hairpins to this palette, we had decided that we disagree with this change, because hairpins _are_ dynamics. However, I understand that this renaming would make life a bit easier for people who have just switched from MS3 or from Finale, where, due to technical considerations and lack of design, hairpins are treated completely differently from other dynamics.

@bkunda What do you think? (The question is whether the "Dynamics" palette should be renamed to "Dynamics & hairpins".)

Reply:
Let's postpone this particular change, since it needs discussion. For the rest, I would like to merge the PR.